crates/telio-network-monitors/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "telio-network-monitors"
+version = "0.1.0"
+edition = "2018"
+license = "GPL-3.0-only"
+repository = "https://github.com/NordSecurity/libtelio"
+publish = false
+
+[dependencies]
+if-addrs.workspace = true
+telio-utils.workspace = true
+tracing.workspace = true
+parking_lot.workspace = true
+tokio = { workspace = true, features = ["full"] }
```

Comment:
Do we need the full featured tokio?

I was wondering if the `unused-features` linter job could be bugged 😅